### PR TITLE
Seeding changes

### PR DESCRIPTION
**Pull request by jeabakker**

### CODE REVIEW

File: docs/intro/elgg-cli.rst [hunk 1]
```diff
@@ -40,13 +40,19 @@ Available commands
     vendor/bin/elgg-cli install [-c|--config CONFIG]
 
     # Seed the database with fake entities
-    vendor/bin/elgg-cli database:seed [-l|--limit LIMIT]
+    # limit: (int) number of items to seed
+    # type: (string) only seed given entity type
+    # create_since: (string) a compatible PHP date/time string to set the lower bound entity time created (eg, '-5 months')
+    # create_until: (string) a compatible PHP date/time string to set the upper bound entity time created (eg, 'yesterday')
+    # image_folder: (string) a folder where the seeder can find images to use as icons, etc.
+    vendor/bin/elgg-cli database:seed [-l|--limit LIMIT] [-t|--type TYPE] [--create_since DATE/TIME] [--create_until DATE/TIME] [--image_folder FOLDER] [create]
```
Comment (jdalsem) [inline docs/intro/elgg-cli.rst]: why not document the `create` flag?

File: engine/classes/Elgg/Database/Seeds/Seed.php [hunk 2]
```diff
@@ -13,17 +13,88 @@
...
+		return elgg_count_entities($options);
+	}
+	
+	/**
+	 * Advance progressbar
+	 *
+	 * @param int $step Step
+	 * @return void
+	 */
+	public function advance($step = 1) {
```
Comment (jdalsem) [inline engine/classes/Elgg/Database/Seeds/Seed.php]: typehint to int... no need to cast on next line

File: engine/classes/Elgg/Database/Seeds/Seeding/TimeHelpers.php [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+	
+	/**
+	 * Set a time for entities to be created after
+	 *
+	 * @param mixed $since Time value for first creation date
+	 *
+	 * @return void
+	 * @see \Elgg\Values::normalizeTimestamp()
+	 */
+	public function setCreateSince($since) {
```
Comment (jdalsem) [inline engine/classes/Elgg/Database/Seeds/Seeding/TimeHelpers.php]: $since could be defaulted to 'now'

**Review by jdalsem — APPROVED** [review]

File: docs/intro/elgg-cli.rst (hunk 1 above)
Comment (jeabakker) [reply docs/intro/elgg-cli.rst]: done

File: engine/classes/Elgg/Database/Seeds/Seed.php (hunk 2 above)
Comment (jeabakker) [reply engine/classes/Elgg/Database/Seeds/Seed.php]: done

File: engine/classes/Elgg/Database/Seeds/Seeding/TimeHelpers.php (hunk 3 above)
Comment (jeabakker) [reply engine/classes/Elgg/Database/Seeds/Seeding/TimeHelpers.php]: done

**Pull request closed without merging**